MoleculeVAE-reImplementation/dataloader.py

```diff
@@ -195,7 +198,7 @@ def caculate_target_values(smiles):
     dataset.
     """
     m = Chem.MolFromSmiles(smiles)
-    logp = Chem.Descriptors.MolLogP(m)
```

Comment:
Calling this way didn't work for my version of `RDkit` but maybe that's my environment.